### PR TITLE
Replaced fwlinks with actual links in csharp/programming-guide/interop/ folder

### PR DESCRIPTION
**Pull request by pkulikov**
Replaced fwlinks with actual links in **csharp/programming-guide/interop** folder to progress #3424 
Tried to replace broken links with new links. 
If no substitution were found, just removed the link (Advanced COM Interoperability in interoperability-overview.md file)

## Suggested Reviewers

@mairaw please review

### CODE REVIEW

File: docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md [hunk 1]
```diff
@@ -42,11 +42,10 @@ The following C# code example illustrates how to use platform invoke services to
...
 ## .NET Framework Security  
- For more information, see [.NET Framework Security](http://go.microsoft.com/fwlink/?LinkId=37122).  
+ For more information, see [.NET Framework Security](https://technet.microsoft.com/en-us/security/).  
   
 ## See Also  
  [C# Programming Guide](../../../csharp/programming-guide/index.md)  
  [Interoperability Overview](../../../csharp/programming-guide/interop/interoperability-overview.md)  
- [Interoperability Overview](../../../csharp/programming-guide/interop/interoperability-overview.md)  
- [A Closer Look at Platform Invoke](http://msdn.microsoft.com/library/ba9dd55b-2eaa-45cd-8afd-75cb8d64d243)  
+ [A Closer Look at Platform Invoke](https://docs.microsoft.com/en-us/dotnet/framework/interop/consuming-unmanaged-dll-functions#a-closer-look-at-platform-invoke)
```
Comment (mairaw) [inline docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md]: good job finding the replacement here. Let's just do a relative link such as:
consuming-unmanaged-dll-functions.md#a-closer-look-at-platform-invoke

**Review by mairaw — CHANGES_REQUESTED** [review]
Your change looks great @pkulikov. Thanks for addressing those fwlinks. Left one small comment to be addressed.

**Comment by pkulikov** [comment]
@mairaw thanks for feedback and a lesson: I didn't know how to reference not pages but parts of them. I've updated the link to the relative one.

**Comment by mairaw** [comment]
Cool @pkulikov! Thanks again.

**Review by mairaw — APPROVED** [review]

**Pull request merged**